src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/perform_interaction_request.cc

```diff
@@ -353,8 +355,14 @@ bool PerformInteractionRequest::ProcessVRResponse(
     return false;
   }
 
+  const hmi_apis::Common_Result::eType code =
+      static_cast<hmi_apis::Common_Result::eType>(
+          message[strings::params][hmi_response::code].asInt());
+  const bool isResultSuccessful = hmi_apis::Common_Result::REJECTED != code;
+
   if (IsInterfaceAwaited(HmiInterfaces::HMI_INTERFACE_UI) &&
-      InteractionMode::MANUAL_ONLY != interaction_mode_) {
+      InteractionMode::MANUAL_ONLY != interaction_mode_ && isResultSuccessful) {
```

Comment:
This looks to be fixing a different issue than #3838. Should there be a separate issue entered for this change?